thunderstorm/src/main/frontend/_ats/ATS_CrudOperations/ui-components.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import {filterDuplicates, ResolvableContent, RuntimeModules, sortArray} from '@nu-art/ts-common';
+import {MandatoryProps_TS_DropDown, TS_DropDown} from '../../components/TS_Dropdown';
+import {SimpleListAdapter} from "../../components/adapter/Adapter";
+import {ModuleFE_BaseApi} from "../../modules/db-api-gen/ModuleFE_BaseApi";
+
+
+export const Props_DBModulesDropDown: ResolvableContent<MandatoryProps_TS_DropDown<ModuleFE_BaseApi<any>>> = {
+    adapter: () => SimpleListAdapter(
+        sortArray(
+            filterDuplicates(
+                RuntimeModules().filter((module: ModuleFE_BaseApi<any>) => !!module.getCollectionName),
+                (module: ModuleFE_BaseApi<any>) => module.getCollectionName()),
+            item => item.getCollectionName()),
+        module => <>{`name: ${module.item.getCollectionName()} | key: ${module.item.getCollectionKey()}`}</>),
```

Comment:
please make the display a bit more clear

<LL_V_L>
{{CollectionName in bold larger letters}}
{{db-key in smaller letters}}
</LL_V_L>